kong/plugins/aws-lambda/schema.lua

```diff
@@ -1,18 +1,22 @@
 return {
   fields = {
-    timeout = {type = "number", default = 60000, required = true },
-    keepalive = {type = "number", default = 60000, required = true },
-    aws_key = {type = "string", required = true},
-    aws_secret = {type = "string", required = true},
-    aws_region = {type = "string", required = true, enum = {
+    timeout = { type = "number", default = 60000, required = true },
```

Comment:
While we appreciate trying to keep style consistent, these whitespace changes here are undesired IMO. This dirties the commit history, as now _all_ the fields in the schema will have been changed with this PR, which does not accurately reflect history. A better approach would be to leave this whitespacing as it, commit your new fields in the proper style, and we can make a separate style cleanup commit in the future.